### ISSUE TITLE
Searching by numeric column

### ISSUE THREAD

**Issue by MikiDylewski** [ppetzold/nestjs-paginate#435]
Hi, recently i started using this package and came by weird issue. What's happening is when i put column `id` in `searchableColumns` and try to search by it like this: `{{baseURL}}?search=145&searchBy=id` i get error
```
QueryFailedError: operator does not exist: integer ~~* unknown
```
I've done some digging and found out that it's because of this line:
```ts
qb.orWhere({
    [column]: ILike(`%${query.search}%`),
})
```
in file `paginate.ts` line 315.
I do also think it's because of that I'm using Postgres. I think that implicit type cast in WHERE clause is either non existent or i have something badly configured
My "fix" was to replace that with:
```ts
qb.orWhere(`${qb.alias}.${column}::VARCHAR ILIKE '%${query.search}%'` )
```
But it's really not good looking.
Thanks in advance

**Comment by ppetzold** [comment]
Duplicate #436

**Comment by ppetzold** [comment]
Fixed with #442
